modules/sno-du-enabling-workload-partitioning.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/sno-du-enabling-workload-partitioning-on-single-node-openshift.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-enabling-workload-partitioning_{context}"]
+= Enabling workload partitioning
+
+Use the following procedure to enable workload partitioning for your single node deployments.
+
+.Procedure
+
+. To enable workload partitioning, you must provide a `MachineConfig` manifest during installation to
+configure CRI-O and kubelet to know about the workload types. The following example shows a manifest, without the encoded file content:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 02-master-workload-partitioning
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+      - contents:
+          source: data:text/plain;charset=utf-8;base64,encoded-content-here
+        mode: 420
+        overwrite: true
+        path: /etc/crio/crio.conf.d/01-workload-partitioning
+        user:
+          name: root
+      - contents:
+          source: data:text/plain;charset=utf-8;base64,encoded-content-here
+        mode: 420
+        overwrite: true
+        path: /etc/kubernetes/openshift-workload-pinning
+        user:
+          name: root
+----
+
+. Provide the contents of `/etc/crio/crio.conf.d/01-workload-partitioning` as the workload partitioning
```

Comment:
Should we mention that this (and the contents of the `openshift-workload-pinning` file below) should be base-64 encoded?  Should we provide an example of how to do that?